console/src/components/Services/Data/TableBrowseRows/ViewActions.js

```diff
@@ -51,34 +60,22 @@ const vSetDefaults = () => ({ type: V_SET_DEFAULTS });
 
 const vMakeRequest = () => {
   return (dispatch, getState) => {
-    const state = getState();
+    const { tables } = getState();
```

Comment:
### Suggestion

I think it would make sense to extract the relevant parts here itself rather than rocking the `tables` object everywhere. This is because, in large schemas, the `tables` object could be **huge**.

You can just:

```js
const {
  currentTable: originalTable,
  currentSchema
} = getState().tables;
```